frontend_lib/src/appContentFactory.js

```diff
@@ -472,6 +536,25 @@ export function appContentFactory (WrappedComponent) {
       return sortTimelineByDate([...timeline, commentForTimeline])
     }
 
+    updateCommentOnTimeline = (comment, timeline, loggedUserUsername) => {
+      const oldComment = timeline.find(timelineItem => timelineItem.content_id === comment.content_id)
```

Comment:
would this work for publications? We would be in frontend/ context where api data are serialized

Reply:
Yes, it works. (sadly) The timeline isn't serialized anywhere.

Reply:
"feature"